README.md

```diff
@@ -5,16 +5,44 @@
 
 `laituri` is a set of Docker-related Python snippets used at [Valohai](https://valohai.com/).
 
+## Usage
+
+### Configuration
+You can configure your used docker command if it is not the default `docker`, using laituri settings.
+_Example:_
+```
+laituri.settings.DOCKER_COMMAND = 'docker'
+```
+
+### Docker credential manager
```

Comment:
`### Docker Credential Manager` for consistency

see other headlines for reference